CoreDataCache+CoreDataProperties.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CoreDataCache+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Dennis Nehrenheim on 19.05.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+extension CoreDataCache {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<CoreDataCache> {
```

Comment:
`CoreDataCache` is `internal` thus `public` modifier will not take any effect.

Suggestion: 
To make the codebase consistent remove `public` modifier

```suggestion
	@nonobjc class func fetchRequest() -> NSFetchRequest<CoreDataCache> {
```